crates/sui-indexer-alt/migrations/2024-10-31-000319_sum_packages/up.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE IF NOT EXISTS sum_packages
```

Comment:
preliminary thought: what if we split out package indexing into two? we can handle system packages with `sum_system_packages`, and otherwise have an append-only `kv_packages`

Reply:
The main reason we don't do this is that we want to be able to efficiently fetch the latest version of any package (system or otherwise) to do type resolution. Seeing as new packages are fairly rare, there's not a lot of value in having that append-only table, and it complicates the read path.